### PR TITLE
Features/16 lightsaber component

### PR DESCRIPTION
**Pull request by rpavlenko**
Add lightsaber timer component with js&scss. 
LightsaberTimer Class has two methods - render() and initTimer(), which you should use to add timer on page and init it.

### CODE REVIEW

**Review by DaveTheWebDev — COMMENTED** [review]
You can use renderElement function to rest of the elements not only section. You can also pass argument attr: {src : '..'} etc

File: styles/components/_LightsaberTimer.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    max-width: 100%;
+    height: 26px;
+    margin-left: 280px;
+    background: #ffffff;
+    box-shadow: 10px -5px 15px $red, 10px 5px 15px $red,
+      inset 0px 5px 4px rgba(255, 0, 0, 0.25);
+    border-radius: 8px;
+    transition: all linear 1s;
+  }
+}
```
Comment (ZibiH) [inline styles/components/_LightsaberTimer.scss:31]: Think about switching to responsive units :)

Comment (rpavlenko) [reply styles/components/_LightsaberTimer.scss:31]: > Think about switching to responsive units :)
Will switch it to responsive units later as we discussed on last meeting.

**Comment by rpavlenko** [comment]
> You can use renderElement function to rest of the elements not only section. You can also pass argument attr: {src : '..'} etc

Done in new commit.

**Review by DaveTheWebDev — APPROVED** [review]
Looks clear

**Pull request merged**